react/admin/CostCenterAddressList.tsx

```diff
@@ -35,34 +45,34 @@ const CostCenterAddressList: React.FC<AddressListProps> = ({
     filterStatements: [],
   }
 
-  const [state, setState] = useState(initialState)
+  const [state, setState] = useState<TableState>(initialState)
   const jsonschema = {
     properties: {
       street: {
         title: formatMessage(messages.address),
-        cellRenderer: ({ rowData }: any) => {
+        cellRenderer: ({ rowData }: CellRendererProps) => {
           return `${rowData.street}, ${rowData.city}, ${rowData.state}, ${rowData.postalCode}`
         },
       },
       receiverName: {
         title: formatMessage(messages.receiverName),
-        cellRenderer: ({ rowData }: any) => {
+        cellRenderer: ({ rowData }: CellRendererProps) => {
           try {
-            const receiver = JSON.parse(rowData?.receiverName)
+            const receiver = JSON.parse(rowData.receiverName)
```

Comment:
Better to keep the nullish
```suggestion
            const receiver = JSON.parse(rowData?.receiverName)
```